### ISSUE TITLE
BMW adding not available in iobroker?

### ISSUE THREAD

**Issue by Harry423** [TA2k/ioBroker.bmw#79]
Hello,

I've got some problems with the new version of the update.
Since end of November  the addon not be able to login at the BMW API anymore.
In other issues I seen that a reinstallation of the adapter will solve this problem.
That's why today I'am deinstalled the instance and deleted the addon.

Now I want to reinstall the adapter I can't found in the repository  anymore ?!

I searched with filter and one by one - nothing found...

![image](https://github.com/user-attachments/assets/071241cc-7635-4a1e-bcc9-18abc2247799)

Is there a problem?

**Comment by TA2k** [comment]
switch to latest repo and reload adapter list

**Comment by Harry423** [comment]
Ok, then I#ve got a lot of errors...

![Screenshot 2024-12-07 at 19 59 36](https://github.com/user-attachments/assets/5d3bbbe3-38b5-45ba-a0fe-23ac46789d03)

**Comment by TA2k** [comment]
2.9.0 is the latest version

**Comment by Harry423** [comment]
Ups :-( I updated the model. Now the connection is already ok...

Thank you very much for the quick support...

**Comment by hombach** [comment]
Shouldn't this issue to be closed?

**Comment by Harry423** [comment]
Yes, I've deinstalled the software and reinstalled it new. After this it works again...
Thank you.

**Comment by mcm1957** [comment]
Please make necessary adaptions to add the adapter to STABLE repository.
LATEST repository contains only BETA software and should be used at production systems only by experienced users.

**Comment by hombach** [comment]
@TA2k  Issue is fixed - adding to stable is handled in other issue - this one should be closed
